### ISSUE TITLE
incorrect reports on missing named export.

### ISSUE THREAD

**Issue by nilzona** [un-ts/eslint-plugin-import-x#205]
the most normal import from `@playwright/test` is 

```
import { expect, test } from "@playwright/test";
```

These are reported as `import-x/named` on `.js` file but they are clearly available for importing.

<img width="972" alt="Screenshot 2024-12-14 at 20 51 40" src="https://github.com/user-attachments/assets/b17d385a-a693-4c43-b287-1cc5b61d7555" />

**Comment by nilzona** [comment]
it probably has to do with something similar like this one. https://github.com/microsoft/playwright/issues/21058
